admin_guide/pruning_resources.adoc

```diff
@@ -339,3 +389,225 @@ Such tags tend to have just one image in its history, which effectively prevents
 them from ever being pruned.
 xref:../dev_guide/managing_images.adoc#tag-naming[Learn more about _istag_
 naming.]
+
+[discrete]
+[[using-secure-connection-against-insecure-registry]]
+==== Using a Secure Connection Against Insecure Registry
+
+If you see a message similar to the following in the output of the `oadm prune
+images`, then your registry is not secured and the `oadm prune images` client
+will attempt to use secure connection:
```

Comment:
"then your registry is not secured and the `oadm prune images` client
will attempt to use secure connection"

should be 

"then your registry is not secured and the `oadm prune images` client
**attempted** to use secure connection"  ?

Reply:
yes